packages/indexer/src/handlers/events/fees/accumulatedFees.event.handler.ts

```diff
@@ -0,0 +1,84 @@
+import { SubstrateEvent } from "@subql/types"
+import { AccumulatedFee } from "@/configs/src/types"
+import { DailyTreasuryRewardService } from "@/services/dailyTreasuryReward.service"
+import { RelayerService } from "@/services/relayer.service"
+import { encodeAddress } from "@polkadot/util-crypto"
+import { wrap } from "@/utils/event.utils"
+import { getBlockTimestamp } from "@/utils/rpc.helpers"
+import { getHostStateMachine } from "@/utils/substrate.helpers"
+import { timestampToDate } from "@/utils/date.helpers"
+
+function formatStateMachine(stateMachineObj: any): string {
```

Comment:
I think we already have a function that does this

Reply:
i can't seem to see this in the repo